dces-drc-integration/src/test/java/uk/gov/justice/laa/crime/dces/integration/client/DrcApiClientTest.java

```diff
@@ -49,15 +48,15 @@ class DrcApiClientTest {
     @Autowired
     private ServicesProperties services;
 
-    @BeforeAll
+    @BeforeEach
```

Comment:
Kicking off the webserver for each call is going to be costly, no? This just to clean out any residual data/modifications? If so is there a lower impact way?

Reply:
I'm guessing in Athar's absence, but it's supposed to be a light-weight operation and isolates any leftover requests on the mock server failing subsequent tests. We can revisit this if it becomes a problem in future.